bazel/python_repositories.bzl

```diff
@@ -37,15 +37,10 @@ def python_repositories(name = ""):
     )
     new_git_repository(
         name = "aioeventlet_repo",
-        remote = "https://github.com/openstack-archive/deb-python-aioeventlet.git",
+        remote = "https://github.com/magma/deb-python-aioeventlet.git",
         build_file = "//bazel/external:aioeventlet.BUILD",
-        commit = "0afed1425fd36139d1a7281475a28fde3349047e",  # Corresponds to: tag = "0.5.1"
-        shallow_since = "1456158219 +0100",
-        patches = [
-            "//lte/gateway/deploy/roles/magma/files/patches:aioeventlet_fd_exception.patch",
-            "//lte/gateway/deploy/roles/magma/files/patches:aioeventlet.py38.patch",
-        ],
-        patch_args = ["-p1"],
+        commit = "86130360db113430370ed6c64d42aee3b47cd619",
+        shallow_since = "1655813728",
```

Comment:
Should this actually be the current datetime or the point of the last commit in the repository?
Do we need it at all, as we do not have (or plan to have) too many branches in this repository?

Reply:
I think it's just an optimization so the point in time needs to be before the latest commit. Maybe a good question for bazel experts.

Reply:
As discussed with Nils online, we may want to change the aioeventlet setup to Bazel in the future, see #13599.